helm/templates/query-service-config.yaml

```diff
@@ -14,7 +14,7 @@ data:
           connectionString = "{{ .Values.queryServiceConfig.data.zookeeperConnectionString }}"
         }
       ]
-      {{- if .Values.handlers }}
```

Comment:
why this check is removed? now this config will have `queryRequestHandlersConfig = []` which will override the queryRequestHandlersConfig in default application.conf file.

Reply:
@ravisingal  i had to remove this since when I tested with saas `helm-values` locally , complete `queryRequestHandlersConfig[]` itself was missing from generated configmap `query-service-config` . I think that is coz `{{- if .Values.handlers }}` returns false for empty array `handlers[]` .  So i removed this if block.
But you are correct this will also override default queryRequestHandlersConfig with empty once. 

Do you have some way using which both the above issues can be addressed ? 
Also I think we might have to provide the default handlers that I removed from this repo `values.yaml` in saas `helm-values` also ...

Reply:
@buchi-busireddy  I tested locally and I dont think 2 `service.config` from different application.conf are getting merged at `queryRequestHandlersConfig`  level.

For example if `queryRequestHandlersConfig` in `/app/resources/configs/common/application.conf` have data  but `queryRequestHandlersConfig` in `/app/resources/configs/query-service/application.conf` is empty , then query-service is starting with empty configuration and not merging both of them.

Reply:
@samarth-gupta-traceable Add the condition as follow:
```
{{- if or .Values.handlers .Values.extraHandlers }}
```

Reply:
@ravisingal  fixed the above as per your suggestion. 

Also created PR to add `handlers` removed from this repo helm chart values.yaml to `helm-values` values.yaml
https://github.com/Traceableai/helm-values/pull/130

we will need to merge above PR first before merging this i guess.
cc @buchi-busireddy